Survey/Configurations/UserDefaults/CachedUserStorage.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CachedUserStorage.swift
+//  Survey
+//
+//  Created by Khanh on 07/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+final class CachedUserStorage: StorageProtocol {
```

Comment:
Why is CachedUserStorage used to store APISurvey?

Reply:
I think its a misleading name, I changed it to SurveyListStorage at https://github.com/nkhanh44/ic-survey-ios/pull/85/commits/d98c5868bbec6d50fa2b3acf6dca5f2de5a39030.